static/app/views/profiling/landing/profileCharts.tsx

```diff
@@ -0,0 +1,145 @@
+import {useMemo} from 'react';
+import {InjectedRouter} from 'react-router';
+import {useTheme} from '@emotion/react';
+
+import {AreaChart} from 'sentry/components/charts/areaChart';
+import ChartZoom from 'sentry/components/charts/chartZoom';
+import {Panel} from 'sentry/components/panels';
+import {PageFilters} from 'sentry/types';
+import {Series} from 'sentry/types/echarts';
+import {axisLabelFormatter, tooltipFormatter} from 'sentry/utils/discover/charts';
+import {useProfileStats} from 'sentry/utils/profiling/hooks/useProfileStats';
+
+interface ProfileChartsProps {
+  query: string;
+  router: InjectedRouter;
+  selection?: PageFilters;
+}
+
+// We want p99 to be before p75 because echarts renders the series in order.
+// So if p75 is before p99, p99 will be rendered on top of p75 which will
+// cover it up.
+const SERIES_ORDER = ['count()', 'p99()', 'p75()'];
+
+export function ProfileCharts({query, router, selection}: ProfileChartsProps) {
+  const theme = useTheme();
+
+  const profileStats = useProfileStats({query, selection});
+  const series: Series[] = useMemo(() => {
+    if (profileStats.type !== 'resolved') {
+      return [];
+    }
+
+    const timestamps = profileStats.data.timestamps;
+
+    const allSeries = profileStats.data.data.map(rawData => {
+      if (timestamps.length !== rawData.values.length) {
+        throw new Error('Invalid stats response');
+      }
+
+      if (rawData.axis === 'count') {
+        return {
+          data: rawData.values.map((value, i) => ({
+            name: timestamps[i] * 1e3,
+            value: value ?? 0,
+          })),
+          seriesName: `${rawData.axis}()`,
+          xAxisIndex: 0,
+          yAxisIndex: 0,
+        };
+      }
+
+      return {
+        data: rawData.values.map((value, i) => ({
+          name: timestamps[i] * 1e3,
```

Comment:
What does name represent here (and why do we multiply it with 1e3), not clear what the intention here is

Reply:
You can see the type definition [here](https://github.com/getsentry/sentry/blob/master/static/app/types/echarts.tsx#L4-L12), the name is basically the `x` axis value in this case.